tests/unit/downloader/test_flatten_concrete_entity.py

```diff
@@ -0,0 +1,55 @@
+import json
+
+from ingest.downloader.flattener import Flattener
+from tests.unit.downloader.test_flattener import FlattenerTest
+
+
+class FlattenConcreteEntityTest(FlattenerTest):
+    def test_flatten__project_metadata(self):
+        self.maxDiff = None
```

Comment:
I don't see it is used anywhere. Is it needed?

Reply:
Ah it's a config var in the test framework. I'll remove it.